weave

```diff
@@ -1478,6 +1564,13 @@ case "$COMMAND" in
                 --rewrite-hosts)
                     REWRITE_HOSTS=1
                     ;;
+                --add-host)
+                    DNS_EXTRA_HOSTS="$2 $DNS_EXTRA_HOSTS"
+                    shift
+                    ;;
+                --add-host=*)
+                    DNS_EXTRA_HOSTS="${1#*=} $DNS_EXTRA_HOSTS"
+                    ;;
```

Comment:
That could be addressed with a `DNS_EXTRA_HOSTS_COUNT`, analogous to `CIDR_ARG_COUNT`